src/environments/environment.ts

```diff
@@ -12,14 +12,17 @@ export const environment = {
       getOne: 'http://localhost:3000/tests'
     },
     activation: {
-      getOne: 'http://localhost:3000/activation'
+      getOne: 'http://localhost:3000/respondent/activation'
     },
     respondents: {
       get: 'http://localhost:3000/admin/respondents',
       save: 'http://localhost:3000/admin/respondents',
     },
     auth: {
-      facebook: 'http://localhost:3000/auth/facebook'
+      facebook: {
+        user: 'http://localhost:3000/auth/facebook',
+        respondent: 'http://localhost:3000/respondent/auth/facebook',
```

Comment:
How it works, respondent routes is protected on back-and, how you can get token by protected route